mitodl/ocw-hugo-themes
# updated about page MIT staff

#### Pre-Flight checklist


#### What are the relevant tickets?
Close #810 

#### What's this PR do?
Updated staff info on about page.

#### How should this be manually tested?
- Start ocw-www
- Visit about page
- Verify all changes mention in issue #810 have been completed

#### Any background context you want to provide?
- The image for Randall Cater is of the wrong aspect ration and can be fixed with stretching but the results are not promising.
- Screenshot of current iteration is attached below
#### Screenshots (if appropriate)
<img width="1440" alt="Screenshot 2022-08-10 at 5 33 02 PM" src="https://user-images.githubusercontent.com/87968618/183903263-f2c58ff6-5a66-4dac-84e5-d7174c654309.png">
